package/CHANGELOG

```diff
@@ -39,6 +39,7 @@ Enhancements
     inputs (PR #4273, part of #3927)
   * Added a warning about charge neutrality to the documentation of
     `DielectricConstant` (Issue #4262, PR #4263)
+  * Added a reader for GROMOS11 TRC trajectories (PR #4292 , Issue #4291)
```

Comment:
I'm not sure the changelog should have an order.  It's the list of changes between two versions, all the changes happen simultaneously, so their order within the changelog isn't important.

Reply:
In principle it doesn't matter, in practice/traditionally we have been  putting the latest changes at the top of the changes in each section. I don't remember where this came from or why we were doing it.

(I know that this reads as a terribly nitpicky comment, my excuse is that I just saw it before starting reading a thesis and happily did something that not thesis-reading to clarify that Hugo is not arbitrarily making the PR more laborious.)

Reply:
Order is very useful when writing release notes - it makes it easier to organise things in a sequence of how the work was done. It's a convenience but it does save a good hour of work during the release to make a suitable blog post.

Reply:
Order is very useful when writing release notes - it makes it easier to organise things in a sequence of how the work was done. It's a convenience but it does save a good hour of work during the release to make a suitable blog post.

Reply:
I put my change now on top. 
I did the in-between positioning earlier to avoid merge conflicts, but I follow the opinion of the majority in this case :)